pandas/tests/io/parser/test_quoting.py

```diff
@@ -38,7 +39,14 @@ def test_bad_quote_char(all_parsers, kwargs, msg):
 @pytest.mark.parametrize(
     "quoting,msg",
     [
-        ("foo", '"quoting" must be an integer|Argument'),
+        (
+            "foo",
+            # This error message changed across versions
+            '"quoting" must be an integer|'
+            + re.escape(
+                "Argument 'quoting' has incorrect type (expected int, got str)"
+            ),
+        ),
```

Comment:
Unrelated with this PR but kept from merge - it seems safer to me than the fix that was recently proposed, what do you think ?

Reply:
I would do this in a follow up PR to reduce the diff.

Reply:
ok